README.md

```diff
@@ -1,3 +1,14 @@
 # Email Scraper
 
-A command line utility to scrape emails from a web page
\ No newline at end of file
+A command line utility to scrape emails from a web page using Python 3.4.3. You can download Python3 [here](https://www.python.org/downloads/). If you're not sure what version of Python you're using, type `$ python --version` or `$ python3 --version` in the terminal.
+
```

Comment:
can you document in here the assumptions we made, and the known limitations as we discussed in the email?